README.md

```diff
@@ -57,7 +57,7 @@ export default class DateTimePickerTester extends Component {
   render() {
     return (
       <>
-        <Button title="Show DatePicker" onPress={this.showDateTimePicker} />
+        <Button title="Show DatePicker" onPress={this.showDateTimePicker()} />
```

Comment:
This doesn't look correct. The function is passed as a reference here and should not be invoked directly.

Reply:
I did try it the suggested way, but the date time picker wouldn't show up. Then I did the change mentioned and it worked!

Also, the documentation says that the model prop is 'datetime' by default right? I wanted both date and time, so I didn't pass in the model prop. But only date showed up. Not sure what the reason exactly is, still figuring out!